share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md

```diff
@@ -35,3 +35,16 @@ To make changes based on feedback:
 Once all feedback has been addressed and if it's a change we want to include (we include most changes) then we'll add your commit to Homebrew. Note it will not show up as "Merged" because of the way we include contributions.
 
 Well done, you are now a Homebrew contributor!
+
+Tips:
+
+* `pushd $(brew --prefix)` before doing any of the following; `popd` afterward.
```

Comment:
Would probably rather recommend using explicit `cd`s here as introducing a new stack command could be a bit confusing.